### DISCUSSION TITLE
Is there a way to see the total # of repos broken down by tier?

### DISCUSSION THREAD

**Discussion by TechStackz** [scottgriv/PRG-Personal-Repository-Guidelines#14] [Q&A] [answered]
hello. I can see there is a total number of repos on the PRG table but I don't see it broken down by tiers. I'd like to see X number of bronze, X number of silver, gold, etc... is this possible?

**Comment by scottgriv** [comment, marked as answer]
Hi! Thank you for taking interest in PRG. At the moment, there's no way to do this unfortunately. It's a constraint around the DataTables Javascript library that PRG uses. I'm only able to show the total at the moment ([see the DataTables documentation here on it](https://datatables.net/reference/option/language.info)), not the tier list broken down. I'll look into this in the future though, I may just add a separate count on the top outside of the data table. 

I actually have this in the [project sheet](https://github.com/users/scottgriv/projects/8?pane=issue&itemId=61178572) as one of my enhancements. You can track the progress of it there and don't hesitate to reach out here if you have anymore questions. Thanks!

**Reply by TechStackz** [reply]
gotcha. thanks!
